core/store/migrations/migrate_test.go

```diff
@@ -285,6 +286,41 @@ func TestMigrate_Migration1565877314(t *testing.T) {
 	assert.Equal(t, "https://unset.url", exiFound.URL.String())
 }
 
+func TestMigrate_Migration1570675883(t *testing.T) {
+	orm, cleanup := bootstrapORM(t)
+	defer cleanup()
+
+	db := orm.DB
+
+	require.NoError(t, migration0.Migrate(db))
```

Comment:
Do we have a way to do all migrations up to 'Migration1570675883'?  It would ensure this test is happening against the latest schema.

Reply:
I'll look into it.

Reply:
I'm personally OK if all migrations are iterated explicitely.  Also, a grep results in some changes which affect the run_results table in other migrations.

Reply:
Done.